_sass/config/_variables.scss

```diff
@@ -51,7 +51,7 @@ $u-space-huge: 10rem;
 
 // fonts
 $bold-font: "Moderat-Bold", sans-serif;
-$body-font: "Muli", sans-serif;
+$body-font: "muli", sans-serif;
```

Comment:
this is just because it follows exactly the copy-and-paste code from Adobe Fonts